src/intl/en/glossary.json

```
@@ -238,7 +238,7 @@
   "message-call-term": "Message call",
   "message-call-definition": "The act of passing a <a href=\"/glossary/#message\">message</a> from one account to another. If the destination account is associated with <a href=\"/glossary/#evm\">EVM</a> code, then the VM will be started with the state of that object and the message acted upon.",
   "mev-term": "MEV",
-  "mev-definition": "A mechanism prioritizing certain actions on a blockchain for a fee, influencing outcomes and the order of transactions.",
+  "mev-definition": "Maximal extractable value (MEV) refers to the maximum value that can be extracted from block production in excess of the standard block reward and gas fees by including, excluding, and changing the order of transactions in a block.",
```

Would remove the term from its definition, and place in `mev-term` instead.

ie. `"mev-term": "Maximal extractable value (MEV)",`

```suggestion
  "mev-definition": "The maximum value that can be extracted from block production in excess of the standard block reward and gas fees by including, excluding, and changing the order of transactions in a block.",
```